slf4cats-api/src/main/scala/slf4cats/api/slf4cats-api.scala

```diff
@@ -3,35 +3,24 @@ package slf4cats.api
 import cats.effect.Sync
 import org.slf4j.Marker
 
-trait LoggingContext[F[_]] {
+trait StructuredLoggerArguments[F[_]] {
```

Comment:
```suggestion
trait ArgumentsBuilder[F[_]] {
```
aaaa, I see now :+1:  So the essence of the trait is that it is a _builder pattern_. I would reflect that in the name